stock_packaging_calculator_packaging_level/models/product_qty_by_packaging_mixin.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# @author: Sébastien Alix <sebastien.alix@camptocamp.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl)
+from odoo import api, models
+
+
+class ProductQtyByPackagingMixin(models.AbstractModel):
+    """Allow displaying product qty by packaging."""
+
+    _inherit = "product.qty_by_packaging.mixin"
+
+    # Amazing.. unlike `api.depends`, `depends_context` cannot use a lambda
+    # to delegate lookup. Hence we are forced to override and call super.
+    @api.depends_context(
+        "lang",
+        "qty_by_pkg_total_units",
+        "qty_by_packaging_type_fname",
+        "qty_by_packaging_type_compact",
```

Comment:
- I don't really see this context being used anywhere. But it's fine if we have something to control it.
- Please rename it appropriately.
```suggestion
        "qty_by_packaging_level_fname",
        "qty_by_packaging_level_compact",
```